src/ApiRequestPanel.js

```diff
@@ -165,13 +164,16 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
   }
 
   _renderServerSelector() {
-    const { amf, hideServerSelector, selectedServerType, selectedServerValue } = this;
+    const { amf, hideServerSelector, selectedServerType, selectedServerValue, serversCount, hideCustomServer } = this;
+    const hidden = (serversCount && serversCount < 1) || !!hideServerSelector;
```

Comment:
I believe this should be getter. Render function (this function is a part of the render function) should not make any computations. Just reflects the current state.